packages/protocol/test/identity/federatedattestations.ts

```diff
@@ -43,8 +48,18 @@ contract('FederatedAttestations', (accounts: string[]) => {
       CeloContractName.FederatedAttestations,
       federatedAttestations.address
     )
-
     initialize = await federatedAttestations.initialize(registry.address)
+
+    await accountsInstance.createAccount({ from: issuer })
+    sig = await getSignatureForAttestation(
+      pnIdentifier,
+      issuer,
+      account,
+      issuedOn,
+      signer,
+      1,
```

Comment:
this may be best fixed in the validate PR but can we make this chainId a constant either within this test file or else directly in the utils file (this is how the MTW tests deal with it)?

Reply:
fixed in validate PR!